deepquest/__init__.py

```diff
@@ -1,12 +1,12 @@
-def train(config=None, changes={}):
+def train(parameters):
     """
     Handles QE model training.
     :param config: Either a path to a YAML or pkl config file or a dictionary of parameters.
     :param dataset: Optional path to a previously built pkl dataset.
     :param changes: Optional dictionary of parameters to overwrite config.
     """
     import deepquest.train
-    train.main(config, changes)
+    train.main(parameters)
```

Comment:
We will need to call `setparameters()` before calling `train.main()`.

Reply:
Do you mean moving the block 7-16 from `__main__.py` to `__init__.py`?

Reply:
Yeah I guess so (see comment above https://github.com/RSE-Sheffield/nmt-keras/pull/39/files#r395495882) - `train.main()` will need to take the parameters dictionary as an argument.

Reply:
OK, makes sense I guess. 
I'll try it out later.

Reply:
done with https://github.com/RSE-Sheffield/nmt-keras/pull/39/commits/b38be88cd4516f7d6b7c1787ece56827596feb6d.